contrib/gorilla/websocket/example_test.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package websocket_test
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gorilla/websocket"
+	muxtrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux"
+	websockettrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/websocket"
+)
+
+// This example illustrates the usage of WrapConn to trace the websocket
+// connections.
+func ExampleWrapConn() {
+	mux := muxtrace.NewRouter()
+
+	var upgrader websocket.Upgrader
+	mux.HandleFunc("/ws", func(w http.ResponseWriter, r *http.Request) {
+		oconn, err := upgrader.Upgrade(w, r, nil)
+		if err != nil {
+			log.Print("upgrade:", err)
```

Comment:
```suggestion
			log.Printf("Error on upgrade: %v", err)
```
Just to be consistent with the other messages in the same example (capital letter at start, %v, etc)

Also, isn't the right approach here to call [`http.Error`](http://golang.org/pkg/net/http#Error) or something similar. We should promote good practices in our examples 🙂